<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [X2Go-Dev] Processed: your mail
   </TITLE>
   <LINK REL="Index" HREF="http://lists.berlios.de/pipermail/x2go-dev/2013-April/index.html" >
   <LINK REL="made" HREF="mailto:x2go-dev%40lists.berlios.de?Subject=Re%3A%20%5BX2Go-Dev%5D%20Processed%3A%20your%20mail&In-Reply-To=%3Chandler.s.C.136696616429899.transcript%40bugs.x2go.org%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="005557.html">
   <LINK REL="Next"  HREF="005559.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[X2Go-Dev] Processed: your mail</H1>
    <B>X2Go Bug Tracking System</B> 
    <A HREF="mailto:x2go-dev%40lists.berlios.de?Subject=Re%3A%20%5BX2Go-Dev%5D%20Processed%3A%20your%20mail&In-Reply-To=%3Chandler.s.C.136696616429899.transcript%40bugs.x2go.org%3E"
       TITLE="[X2Go-Dev] Processed: your mail">owner at bugs.x2go.org
       </A><BR>
    <I>Fri Apr 26 10:49:29 CEST 2013</I>
    <P><UL>
        <LI>Previous message: <A HREF="005557.html">[X2Go-Dev] Bug#194: Traceback in pyhoca-gui
</A></li>
        <LI>Next message: <A HREF="005559.html">[X2Go-Dev] Processed: your mail
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#5558">[ date ]</a>
              <a href="thread.html#5558">[ thread ]</a>
              <a href="subject.html#5558">[ subject ]</a>
              <a href="author.html#5558">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Processing commands for <A HREF="https://lists.berlios.de/mailman/listinfo/x2go-dev">control at bugs.x2go.org</A>:

&gt;<i> reassign #194 python-x2go
</I>Bug #194 [pyhoca-gui] Traceback in pyhoca-gui
Bug reassigned from package 'pyhoca-gui' to 'python-x2go'.
Ignoring request to alter found versions of bug #194 to the same values previously set
Ignoring request to alter fixed versions of bug #194 to the same values previously set
&gt;<i> found #194 0.4.0.2
</I>Bug #194 [python-x2go] Traceback in pyhoca-gui
There is no source info for the package 'python-x2go' at version '0.4.0.2' with architecture ''
Unable to make a source version for version '0.4.0.2'
Marked as found in versions 0.4.0.2.
&gt;<i> thanks
</I>Stopping processing here.

Please contact me if you need assistance.
-- 
194: <A HREF="http://bugs.x2go.org/cgi-bin/bugreport.cgi?bug=194">http://bugs.x2go.org/cgi-bin/bugreport.cgi?bug=194</A>
X2Go Bug Tracking System
Contact <A HREF="https://lists.berlios.de/mailman/listinfo/x2go-dev">owner at bugs.x2go.org</A> with problems
</PRE>



<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="005557.html">[X2Go-Dev] Bug#194: Traceback in pyhoca-gui
</A></li>
	<LI>Next message: <A HREF="005559.html">[X2Go-Dev] Processed: your mail
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#5558">[ date ]</a>
              <a href="thread.html#5558">[ thread ]</a>
              <a href="subject.html#5558">[ subject ]</a>
              <a href="author.html#5558">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.berlios.de/mailman/listinfo/x2go-dev">More information about the X2Go-Dev
mailing list</a><br>
</body></html>
